dali/base/dautils.cpp

```diff
@@ -515,7 +523,17 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    normalizeName(name, lfn, false);
+    if (isExternalFile(name))
+    {
+        lfn.set(name);
+        lfn.toLowerCase();
+        const char *ns1 = strstr(name,"::>");
```

Comment:
This will have a subtly different effect from the previous code.  E.g., on file filename

file::ip::one::>two

I think it may require @jakesmith 's input to know what this code is there for, and whether it needs preserving.